Source/Core/DolphinQt/GCMemcardManager.cpp

```diff
@@ -47,6 +49,17 @@ GCMemcardManager::GCMemcardManager(QWidget* parent) : QDialog(parent)
 
   m_timer->start(1000 / 8);
 
+  for (int i = 0; i < SLOT_COUNT; i++)
```

Comment:
This should probably be placed into its own self-descriptive function and called here, like whats done to separate widget creation, widget connections, etc above.